ui/src/dashboards/components/TemplateVariableRow.js

```diff
@@ -30,7 +32,9 @@ const RowValues = ({
   onStartEdit,
   autoFocusTarget,
 }) => {
-  const _values = values.map(({value}) => value).join(', ')
+  const _values = uniq(values.map(({value}) => value))
```

Comment:
I like the `v => v.value` pattern as well. Just got trigger happy and then realized I didn't make that change 😕